src/initdt.jl

```diff
@@ -35,14 +35,14 @@ function ode_determine_initdt{tType,uType}(u0,t::tType,tdir,dtmax,abstol,reltol,
 
   #@. u₁ = @muladd u0 + tdir*dt₀*f₀
   @tight_loop_macros for i in uidx
-    @inbounds u₁[i] = u0[i] + tdir*dt₀*f₀[i]
+    @inbounds u₁[i] = u0[i] + (tdir*dt₀)*f₀[i]
```

Comment:
probably best to just hoist that out of the loop. Does `@muladd` work on this now? I think it had a problem with it before.